src/main/java/net/sf/jabref/logic/error/StreamEavesdropper.java

```diff
@@ -33,40 +34,47 @@ public static StreamEavesdropper eavesdropOnSystem() {
         return streamEavesdropper;
     }
 
-    public TeeStream getOutStream() {
-        PrintStream consoleOut = new PrintStream(outByteStream) {
+    /**
+     * Return a new {@code PrintStream} which also creates a new log event with {@link Level#WARN} for each message and forwards it to the {@link LogMessages} archive.
+     * @return a PrintStream
+     */
+    public PrintStream getOutStream() {
+        return new PrintStream(systemOut)  {
             @Override
             public void write(byte[] buf, int off, int len) {
                 super.write(buf, off, len);
                 String message = new String(buf, off, len);
-                addToLog(message, MessageType.OUTPUT);
+                addToLog(message, Level.WARN);
             }
         };
-        return new TeeStream(consoleOut, systemOut);
     }
 
-    public TeeStream getErrStream() {
-        PrintStream consoleErr = new PrintStream(errByteStream) {
+    /**
+     * Return a new {@code PrintStream} which also creates a new log event with {@link Level#ERROR} for each message and forwards it to the {@link LogMessages} archive.
+     * @return a PrintStream
+     */
+    public PrintStream getErrStream() {
+        return new PrintStream(systemErr) {
             @Override
             public void write(byte[] buf, int off, int len) {
                 super.write(buf, off, len);
                 String message = new String(buf, off, len);
-                addToLog(message, MessageType.EXCEPTION
-                );
+                addToLog(message, Level.ERROR);
             }
         };
-        return new TeeStream(consoleErr, systemErr);
     }
 
-    private void addToLog(String s, MessageType priority) {
-        if (!s.equals(System.lineSeparator())) {
-            LogMessage messageWithPriority = new LogMessage(s.replaceAll(System.lineSeparator(), ""), priority);
+    /**
+     * Creates a new log event with the given parameters and forwards it to the {@link LogMessages} archive.
+     * @param message message of log event
+     * @param priority level of log event
+     */
+    private void addToLog(String message, Level priority) {
```

Comment:
Please go through the code that all `priority` and `prio` are properly renamed to `level`.